03.bowling/bowling.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+scores = ARGV[0].split(',')
+
+shots = []
+NUMBER_IN_LASTFRAME = 19
+scores.each do |shot|
+  if shot == 'X'
+    shots << 10
+    shots << 0 if shots.size < NUMBER_IN_LASTFRAME
+  else
+    shots << shot.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+def strike?(frame)
+  frame[0] == 10
+end
+
+def spare?(frame)
+  frame.sum == 10
```

Comment:
```suggestion
  !strike?(frame) && frame.sum == 10
```

間違ってストライクをスペアと判定しないようにこうしておくといいかもしれません

Reply:
なるほど...そのままだとストライク[10,0]の場合が除外できていませんでした。修正いたしました。